modules/salt/pages/yomi.adoc

```diff
@@ -0,0 +1,174 @@
+[[yomi.installer]]
+= Install with Yomi
+
+Yomi (yet one more installer) is an installer for openSUSE operating systems.
+It is a SaltStack state, and can be used for installing Salt clients.
+
+Yomi consists of two components:
+
+* The Yomi formula, which contains the Salt states and modules required to drive the installation.
+For the formula, see https://build.opensuse.org/package/show/systemsmanagement:yomi/yomi-formula.
+* The LiveCD operating system image, which includes the `pre-configured ``salt-minion`` service.
+For the image, see https://build.opensuse.org/package/show/systemsmanagement:yomi/openSUSE-Tumbleweed-Yomi
+
+
+To use Yomi for installing your client operating system, you need to follow this process:
+
+* Install and configure the ``salt-master``service.
+* Install the Yomi formula package.
+* Prepare the Salt pillar for the new installation.
+* Boot the new systems with the operating system image, or use PXE boot.
+
+For more information about using Yomi, see https://github.com/openSUSE/yomi/blob/master/README.md
+
+
+
+== Install and Configure salt-master
+
+You will require the ``salt-master`` service to be able to use Yomi for your client operating system.
+If you have not yet installed it on the {productname} Server, you will need to do this step first.
+
+
+
+.Procedure: Installing salt-master
+
+. On the {productname} Server, at the command prompt, as root, install the ``salt-master`` service:
++
+----
+zypper in salt-master
+----
+. Use ``systemctl`` to enable the service:
++
+----
+systemctl enable --now salt-master.service
+----
+
+Yomi uses a Salt pillar to provide configuration information for the installation.
+This can include the layout of the hard disks, the software
+patterns to install, or the users to create.
+
+By default, Salt searches for states in the [path]``/srv/salt/`` directory.
+By default, Salt searches for pillars in [path]``/srv/pillar``.
+
+You do not need to perform this step, if you are using the Yomi formula.
+
+
+.Procedure: Configuring salt-master
+
+. On the {productname} Server, at the command prompt, as root, create two new directories for the configuration file:
++
+----
+mkdir -p venv/etc/salt/pki/{master,minion}
+----
+.  Create two new virtual environments:
```

Comment:
Because this section is part of the salt-master installation, and also not required in Uyuni context, it will not appear at the end. Only to mention that never deploy anything Python-related via a virtual environment : )))